src/main/java/Task.java

```diff
@@ -0,0 +1,57 @@
+import java.util.*;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Task {
+    private static List taskList = new LinkedList();
+    public static void addList(String task){
```

Comment:
It would be good to add white spaces and new lines as stated in the coding standard. I like the implementation, it is easy to follow

Reply:
an example is adding a whitespace between the opening bracket and the parenthesis in a method definition (deleteList(String task) {)